tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/OnBehalfOfTests.cs

```diff
@@ -220,11 +220,11 @@ public async Task OboAndClientCredentials_WithRegional_ReturnsCorrectTokens_Test
             Assert.AreEqual(TokenSource.Cache, clientResult.AuthenticationResultMetadata.TokenSource);
         }
 
-        [RunOn(TargetFrameworks.NetCore)]
+        [RunOn(TargetFrameworks.NetFx)]
         public async Task WithMultipleUsers_TestAsync()
         {
             var aadUser1 = (await LabUserHelper.GetDefaultUserAsync().ConfigureAwait(false)).User;
-            var aadUser2 = (await LabUserHelper.GetAdfsUserAsync(FederationProvider.AdfsV2, true).ConfigureAwait(false)).User;
+            var aadUser2 = (await LabUserHelper.GetDefaultUser2Async().ConfigureAwait(false)).User;
```

Comment:
ah, we never needed an ADFS user for this test. We should make it a point to always rely on cloud users for our tests unless if it is ADFS specific.